mmv1/products/workstations/api.yaml

```diff
@@ -447,3 +447,114 @@ objects:
               description: |
                 A list of messages that carry the error details.
               item_type: Api::Type::KeyValuePairs
+  - !ruby/object:Api::Resource
+    name: "Workstation"
+    self_link: "projects/{{project}}/locations/{{location}}/workstationClusters/{{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}/workstations/{{workstation_id}}"
+    base_url: "projects/{{project}}/locations/{{location}}/workstationClusters/{{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}/workstations"
+    create_url: "projects/{{project}}/locations/{{location}}/workstationClusters?workstationClusterId={{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}/workstations/{{workstation_id}}"
+    update_verb: :PATCH
+    update_mask: true
+    min_version: beta
+    description: "A single instance of a developer workstation with its own persistent storage."
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        "Workstations": "https://cloud.google.com/workstations/docs/"
+      api: "https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters.workstationConfigs.workstations/create"
```

Comment:
```suggestion
      api: "https://cloud.google.com/workstations/docs/reference/rest/v1beta/projects.locations.workstationClusters.workstationConfigs.workstations"
```

Reply:
I actually have a trailing `/create` in all other workstation resources as well, do they need to be removed there as well?

Reply:
Removed trailing `/create`, still present on the existing workstation resources though.

Reply:
Overview's _slightly_ preferred over /create, but probably not worth going back and changing.

Reply:
Check, I'll just leave it as is for now.